beacon-chain/sync/data_columns.go

```diff
@@ -36,9 +63,8 @@ import (
 //   - returns on success when all the initially missing sidecars in `dataColumnsToFetch` are retrieved, or
 //   - returns an error if all peers in `peers` are exhausted and at least one data column sidecar is still missing.
 //
-// TODO: In case at least one column is still missing after peer exhaustion,
-//
-//	but `peers` custody more than 64 columns, then try to fetch enough columns to reconstruct needed ones.
+// In case at least one column is still missing after peer exhaustion,
```

Comment:
Could you move this part of the comment just after `-In case of peers unable to actually provide...`?

Reply:
705b4c70f1f0cf244751ee7f7633a3b7b0b1cae8